DataFormats/PortableTestObjects/src/classes_def.xml

```diff
@@ -6,7 +6,7 @@
     <field name="y_" comment="[nElements_]"/>
     <field name="z_" comment="[nElements_]"/>
     <field name="id_" comment="[nElements_]"/>
-    <field name="metadata()" comment="!"/>
```

Comment:
@ericcano @makortel 
I'm not sure why we used a rule to mark a _method_ as transient. From the tests, I do not see any issues if I drop it.

Reply:
This is a leftover from investigating ROOT serialization issues, and can be removed.